src/Integration/FilterExceptionEvent.php

```
@@ -0,0 +1,32 @@
+    public function __construct(
+        private array $exceptions
+    )
+    {
+    }
+
+    public function getExceptions(): array
+    {
+        return $this->exceptions;
+    }
+
+    public function setExceptions(array $exceptions): void
```

PHPStan (2.1.15)

Method Frosh\SentryBundle\Integration\FilterExceptionEvent::setExceptions() has parameter $exceptions with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)